[2024-05-24T12:01:55.222+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-24T12:01:55.230+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: upload_from_github_to_s3.download_and_upload_to_s3 scheduled__2024-03-01T00:00:00+00:00 [queued]>
[2024-05-24T12:01:55.234+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: upload_from_github_to_s3.download_and_upload_to_s3 scheduled__2024-03-01T00:00:00+00:00 [queued]>
[2024-05-24T12:01:55.234+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-24T12:01:55.240+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): download_and_upload_to_s3> on 2024-03-01 00:00:00+00:00
[2024-05-24T12:01:55.243+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=976) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-24T12:01:55.244+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'upload_from_github_to_s3', 'download_and_upload_to_s3', 'scheduled__2024-03-01T00:00:00+00:00', '--job-id', '626', '--raw', '--subdir', 'DAGS_FOLDER/upload_from_github_to_s3_dag.py', '--cfg-path', '/tmp/tmp4_xb5i3y']
[2024-05-24T12:01:55.245+0000] {standard_task_runner.py:63} INFO - Started process 983 to run task
[2024-05-24T12:01:55.246+0000] {standard_task_runner.py:91} INFO - Job 626: Subtask download_and_upload_to_s3
[2024-05-24T12:01:55.264+0000] {task_command.py:426} INFO - Running <TaskInstance: upload_from_github_to_s3.download_and_upload_to_s3 scheduled__2024-03-01T00:00:00+00:00 [running]> on host b2ab95bd73fd
[2024-05-24T12:01:55.300+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='upload_from_github_to_s3' AIRFLOW_CTX_TASK_ID='download_and_upload_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-01T00:00:00+00:00'
[2024-05-24T12:01:55.300+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-24T12:01:55.705+0000] {credentials.py:1147} INFO - Found credentials in environment variables.
[2024-05-24T12:01:55.768+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-24T12:01:55.769+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-24T12:01:55.773+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=upload_from_github_to_s3, task_id=download_and_upload_to_s3, run_id=scheduled__2024-03-01T00:00:00+00:00, execution_date=20240301T000000, start_date=20240524T120155, end_date=20240524T120155
[2024-05-24T12:01:55.794+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-24T12:01:55.801+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-24T12:01:55.802+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
